### ISSUE TITLE
Cannot find module "wrappers/md"

### ISSUE THREAD

**Issue by fson** [gatsbyjs/gatsby#43]
Steps to reproduce:
```
npm install -g gatsby@0.4.2
gatsby new gatsby-test
cd gatsby-test
gatsby build
```

```
Generating static html pages
failed at generating static html pages
{ [Error: Cannot find module "wrappers/md"] code: 'MODULE_NOT_FOUND' }
```

**Comment by fson** [comment]
Shouldn't this be a relative path `./wrappers/md`? I wonder how it worked before. Does CoffeeScript handle imports differently?

**Comment by KyleAMathews** [comment]
The problem is we switched the wrappers to use .jsx and our babel-loader config is currently excluding modules passing the regex `/node_modules/`. Which is fine when I'm developing locally but when installed, the path of the wrappers is `/usr/local/lib/node_modules/gatsby/lib/isomorphic/wrappers/` which means they're not getting compiled with JSX which means they're throwing an error (the errors are easier to see if you do `gatsby serve` and look in your browser console).

Trying now to fix the regex to *not* exclude Gatsby lib files.

**Comment by KyleAMathews** [comment]
So it turns out you can use a function for the exclude check on loaders which proved a lot simpler than trying to write a gnarly regex that would only exclude the "right" `node_module` directories.

Released a 0.4.3 with the fix.

Thanks for the bug report!

**Comment by KyleAMathews** [comment]
Also the reason things worked with cjsx/coffeescript before was their loaders don't exclude `node_modules` files like babel does.
